README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
would you kindly answer the questions? ignoring the points i made and keep pushing your opinions is not a discussion.

you are suggesting rather lots of changes in all the repositories.